Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -85,13 +85,19 @@ public virtual void Awake()
             InitializeSingleton();
 
             // headless mode? then start the server
-            if (Utils.IsHeadless() && startOnHeadless)
+            if (IsHeadless() && startOnHeadless)
             {
                 Application.targetFrameRate = 60;
                 StartServer();
             }
         }
 
+        // headless mode detection
```

Comment:
this should have been in a separate PR. there is no reason why IsHeadless should be in NetworkManager - it doesn't manage any networking.